### PR TITLE
# Provides Command to remove orphaned databases

### PR DESCRIPTION
**Pull request by demichl68**

### CODE REVIEW

File: src/Console/Command/CleanBranchCommand.php [hunk 1]
```diff
@@ -60,7 +62,7 @@ protected function execute(InputInterface $input, OutputInterface $output)
             'jiraPassword' => $config['CleanBranch']['password']
         ]));
 
-        $io->title('Scan folder ' . $folder . ' for outdated issues');
```
Comment (TuningYourCode) [inline src/Console/Command/CleanBranchCommand.php]: you sure?

Comment (demichl68) [reply src/Console/Command/CleanBranchCommand.php]: Defo!
That's the CleanBranch Command which purges orphaned folders. ;)

File: src/Console/Command/CleanMySQLDatabaseCommand.php [hunk 2]
```diff
@@ -0,0 +1,178 @@
...
+        )) {
+            throw new \RuntimeException('MySQL config is incorrect! Username password and hostname required!');
+        }
+
+        $dbUser = $config['MySQL']['username'];
+        $dbPassword = $config['MySQL']['password'];
+        $dbHost = $config['MySQL']['hostname'];
+
+        $issueService = new IssueService(new ArrayConfiguration([
+            'jiraHost' => $config['CleanBranch']['hostname'],
```
Comment (demichl68) [inline src/Console/Command/CleanMySQLDatabaseCommand.php:87]: Config 'CleanBranch' should be more revealing as it is used in both, CleanBranch and CleanMySQL Commands.

File: src/Console/Command/CleanBranchCommand.php (hunk 1 above)
Comment (demichl68) [reply src/Console/Command/CleanBranchCommand.php]: Oh wow.... them tomatoes! Now I see it 👍

**Pull request merged**